dask_sql/physical/rex/core/call.py

```diff
@@ -46,6 +46,8 @@ def as_timelike(op):
         return np.datetime64(op)
     elif pd.api.types.is_datetime64_dtype(op):
         return op
+    elif isinstance(op, np.timedelta64):
+        return op
```

Comment:
we can probably combine the two conditions since they return the same result.
```suggestion
    elif pd.api.types.is_datetime64_dtype(op) or isinstance(op, np.timedelta64):
        return op
```